packages/mint-components/src/components/sqm-popup-container/sqm-popup-container.tsx

```diff
@@ -15,9 +15,9 @@ import { getProps } from "../../utils/utils";
 })
 export class PopupContainer {
   /**
-   * Show Powered by Impact.com link
+   * Hide Powered by Impact.com link
    *
-   * @uiName Show powered by
+   * @uiName Hide powered by
```

Comment:
The label was reversed without any of the logic being reversed, this is causing it to show up when "Hide Powered By" is true
it needs to stay as "Show Powered By""
(same problem in vanilla-components PR)